lib/plugins/features/movies/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const Controller     = require('./controller');
+const MovieValidator = require('../../../validators/movie');
+
+exports.register = (server, options, next) => {
+
+  server.route([{
+    method: 'POST',
+    path: '/movies',
```

Comment:
Not strictly necessary, but a decent habit to get into: Adding `tags` here improves readability by easily showing what endpoint matches what controller. I'd probably add a 
`tags: ['create', 'movies']` here like seen [here](https://github.com/lob/lob-api/blob/master/src/plugins/features/template-versions/index.js#L20)